yapapi/executor/__init__.py

```diff
@@ -186,6 +188,10 @@ async def _worker(
         ) -> None:
             nonlocal job
 
+            if self._implicit_init:
+                work_context.deploy()
+                work_context.start()
```

Comment:
These two calls add commands to the internal `Script` instance kept by a `WorkContext`. When the non-deprecated way of creating scripts is used (i.e. calling `new_script` on the `WorkContext` and adding steps to that `Script`) in the requestor application, these commands will end up never being called.
One option I can think of is to perform the implicit init in `WorkContext#new_script` and use an internal bool flag to check whether deploy/start should be added to the newly returned `Script` instance.

Reply:
:(

Reply:
Fixed in bc86bae, hopefully.

Reply:
I've chosen other solution than proposed because now everything is still in Executor, without any implicit-init related things in WorkContext and I think there's some value in having this in a single place.